TUTORIAL.md

```diff
@@ -700,16 +709,24 @@ Now it's time for one of the coolest places on the internet - *production!*
 
 Before you deploy these applications to production, I recommend stopping both. That way there's no resource contention for file access. 
 
-To deploy this app to Cloud Foundry, you'll need 1) to update `deploy.sh` in the root directory to set environment variables, 2) set the URL of the server in a different file, and 3) add `pushstate: enabled` to  `Staticfile`. You can see the [modified deploy.sh on GitHub](https://github.com/oktadeveloper/okta-spring-boot-angular-pwa-example/blob/master/deploy.sh).
+To deploy this app to Cloud Foundry, you'll need to 
 
-// todo: IMO, this note will look better if it has a box around it, like a callout of sorts
+1. Update `deploy.sh` in the root directory to set environment variables
+2. Set the URL of the server in a different file
+3. Add `pushstate: enabled` to  `Staticfile`. 
+
+You can see the [modified deploy.sh on GitHub](https://github.com/oktadeveloper/okta-spring-boot-angular-pwa-example/blob/master/deploy.sh).
+
+// todo: IMO, this note will look better if it has a box around it, like a callout of sorts -- agree
 **NOTE:** An alternative to enabling push state is to use hashes in the URL. To do this, you can pass in `{useHash: true}` when creating your routes. 
           
 ```typescript
 RouterModule.forRoot(appRoutes, {useHash: true})
 ```
 
-Unfortunately, this causes the redirect login button to fail since there will be two hashes in the URL.
+Unfortunately, this causes the redirect login button to fail since there are now two hashes in the URL.
+
+// Is the stuff below the solve for that line above?
```

Comment:
I don't think so off the top of my head. This page seems to have something like what I'm looking for, but it's a twitter widget. https://developer.okta.com/use_cases/mfa/